RRS/staff/tests.py

```diff
@@ -1,3 +1,87 @@
+from django.contrib.auth.models import User
+from django.core.urlresolvers import resolve, reverse
 from django.test import TestCase
+from django.test.client import Client
 
-# Create your tests here.
+from base import models as base_models
+from helpers import LinkSanityTestCase
+from staff.forms import RestaurantForm
+
+
+def get_user_model():
```

Comment:
So that when we integrate OAuth, it is easier to refactor in case we use a different user model. Not relevant now though.